routes/task.go

```diff
@@ -0,0 +1,113 @@
+//Package routes -> THIS FILE WAS JUST AN EXERCISE STILL INCLUDED TO SHOW THE WORK MADE ON LOOPS THAT I DIDNT HAVE ON THE NORMAL REQUESTS
```

Comment:
You ended up using `for`s [here](https://github.com/wizelineacademy/golang-bootcamp-2020/pull/25/files#diff-102d688356f8f617b5a2cc05d9989ddd74ac87a17036fe50a617890849cbf587R70), so I feel this can be deleted.